server/controllers/users.js

```diff
@@ -10,15 +10,15 @@ module.exports.signin = async (req, res) => {
   try {
     const existingUser = await User.findOne({ email });
     if (!existingUser)
-      return res.status(404).json({ message: "User doesn't exists." });
+    return res.status(404).send("User doesn't exists!");
```

Comment:
I recommend using this package for setting status codes:
https://www.npmjs.com/package/http-status-codes

That way you can set some consistency with status codes. This would then look something like
`    return res.status(StatusCodes.NOT_FOUND).send("User doesn't exists!");`